src/extensions/default/bramble/lib/launcher.js

```diff
@@ -28,7 +28,14 @@ define(function (require, exports, module) {
         var server = this.server;
         var browser = this.browser;
 
-        browser.update(server.serveLiveDoc(url));
+        server.serveLiveDoc(url, function(err, urlOrHTML) {
+            if(err) {
+                // TODO: how to deal with this error?                
+                console.error("[Launcher Error]", err);
```

Comment:
The important thing is that the logging is helpful, unless we also want users to be able to report errors. In that case, we could serve an error document with details on what went wrong.

Otherwise, maybe some more intelligent error parsing?